docs/navigation-configuration.md

```diff
@@ -1,242 +1,264 @@
 
-- `https://luigiexample.com/projects/pr1` defines the `projects/pr1` navigation path. It is not a valid navigation path, since a valid navigation path always starts from the root.
+For more information on setting caching with view refreshing and preloading for view groups, read [this document](navigation-parameters-reference.md#node-parameters).
 
+## Creating a dynamic path
+In Luigi, you can make a navigation path dynamically changeable according to your needs. This is accomplished by defining variables within the **pathSegement** or **viewUrl** navigation paths. 
 
-## Path parameters
+### pathSegment variables
+Instead of a static value for your **pathSegment**, you can add a colon to this value to make it act as a variable, for example `:userId`. This tells Luigi to accept any value for this **pathSegment**. 
 
-Use path parameter values to define the **pathSegment** in your configuration. You can either use a static value for your **pathSegment**, or add a colon to this value as in `:projectId`, to make it act as a parameter. This tells Luigi to accept any value for this **pathSegment** of the main application URL. The value replaces the parameter when it is further processed by the application.
+In this example, a `userId`path variable is defined: 
 
-A sample structure with a parametrized **pathSegment** is as follows:
-```
-{
-  navigation: {
+````javascript
+navigation: {
     nodes: [
       {
         pathSegment: 'home',
         label: 'Home',
-        viewUrl: 'https://my.microfrontend.com/',
+        viewUrl: 'https://microfrontend.com/',
         children: [
           {
-            pathSegment: 'projects',
-            label: 'Projects',
-            viewUrl: 'https://my.microfrontend.com/projects/list.html',
+            pathSegment: 'users',
+            label: 'User List',
+            viewUrl: 'https://microfrontend.com/users/list.html',
             children: [
               {
-                pathSegment: ':projectId',
-                label: 'Project Details',
-                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=:projectId;'
-              }            
-			]
+                pathSegment: ':userId',
+                label: 'User Profile',
+                // E.g. if userId is 'JohnSmith'
+                // the main application URL will be https://yourwebsite.com/users/JohnSmith
+              }
+            ]
           }
         ]
       }
     ]
   }
 }
+...
+````
 
-```
-
- Use the following options to work with path parameters:
+### viewUrl variables
 
-- Add the parameters to the **viewUrl** by placing them anywhere in the **viewUrl** value. For example, if the main application URL is `https://luigiexample.com/home/projects/pr23`, then the **viewUrl** of the micro frontend in the content area is `https://my.microfrontend.com/projects/details.html#id=pr23`. 
+You have three options to add a variable to **viewUrl**: 
```

Comment:
```suggestion
You have the following options to add a variable to **viewUrl**: 
```
I was once told that it is better to write `the following` than a specific number, in case the number changes. :)